nzedb/Binaries.php

```diff
@@ -1027,16 +1026,16 @@ protected function storeHeaders(array $headers, $multiGroup)
 		$binariesQuery = rtrim($binariesQuery, ',') . $binariesEnd;
 
 		// Check if we got any binaries. If we did, try to insert them.
-		if (((strlen($binariesCheck . $binariesEnd) === strlen($binariesQuery)) ? true : $this->_pdo->queryExec($binariesQuery))) {
+		if ((strlen($binariesCheck . $binariesEnd) === strlen($binariesQuery)) ? true : $this->_pdo->queryExec($binariesQuery)) {
```

Comment:
We shouldn't use ternaries inside if conditions.

Reply:
As in previous comment, i left it as it was, just removed unnecessary brackets.

Reply:
If you're correcting style in massive blocks like this, it's probably best to correct it all in one go.